packages/vue-ds/src/components/Pagination/Pagination.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <section v-if="!isInvalid" :class="PaginationClass">
+    <div :class="PaginationContainerClass">
+      <button
+        v-if="hasPreviousCondition"
+        class="paginationButton"
+        :disabled="isLoading"
+        @click="prevPage"
+      >
+        <AngleLeft :width="12" :height="23" :viewBox="'-1 3 12 7'" />
+      </button>
+      <div
+        v-for="(currentPage, idx) in range"
+        :key="idx"
```

Comment:
as `currentPage` is a primitive data, you can use it at `:key` and remove `idx`. What do you think?

Reply:
you're totally right! thanks 😄

Reply:
@tassioFront is really a monster 😁

Reply:
😬😬😬😬❤️❤️